vortex-array/src/array/constant/compute/mod.rs

```diff
@@ -48,6 +52,30 @@ impl ComputeVTable for ConstantEncoding {
     }
 }
 
+impl BinaryNumericFn<ConstantArray> for ConstantEncoding {
```

Comment:
I know it's annoying, but where we've already converted `compute` to be a module, can we try to put each compute function in its own file?